README.md.tt

```diff
@@ -61,7 +61,7 @@ it first to leverage the `cache_from` settings of Docker Compose which avoids re
 * Build the Docker image:
 
 ```sh
-docker-compose -f docker-compose.test.yml build
+./bin/docker-prepare && docker-compose -f docker-compose.test.yml build
```

Comment:
How about moving `./bin/docker-prepare` to something like `Prerequisite before run test using docker or build the docker`? because
1. This command can run only once after setup the template as it just create the directory.
2. If user run test without running the build step. It will still raise the error.

What do you think @andyduong1920 ?

Reply:
it's for the build step @gutakk so it could be something like `Prerequisite to build the docker image`, do you have any idea for that name?

Reply:
How about put it under the `* Build the Docker image:` section something like this
```suggestion
Before run the build command, run following command once to prepare the docker
```
```sh
./bin/docker-prepare
```

You can rearrange the instruction above.

Reply:
it needs to run every time we build the image @gutakk